services/entitlement/service/README.md

```diff
@@ -83,7 +83,7 @@ The `yarn setup:webhooks` script is used for development environments. To get
 the values for `ENTITLEMENT_WEBHOOK_SECRET` and `MANIFEST_WEBHOOK_SECRET` env
 variables for the production environment, the following mutation must be
 launched against the Video Service GraphQL API. You can use GraphiQL UI for that
-(https://encoding.service.eu.axinom.net/graphiql):
+(https://video.service.eu.axinom.net/graphiql):
```

Comment:
This URL is currently not accessible. Need to make sure it is before this PR is completed

Reply:
Wrote with Anushka and he said the change is in the repo and once it is pushed the URL will work.